phinx.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+use OpenCFP\Kernel;
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$kernel = new Kernel((string) \OpenCFP\Environment::fromServer($_SERVER), false);
```

Comment:
What do you think about importing `OpenCFP\Environment` in order to avoid unnecessary FQCNs?